### DISCUSSION TITLE
New to Fooocus

### DISCUSSION THREAD

**Discussion by persius77** [lllyasviel/Fooocus#2697] [Q&A] [answered]
Hello
As a new user (2 weeks), I've been experimenting with Fooocus and have run into some issues that so far I have not been able to find answers for so I hope some experienced users will be able to assist me.    I will be as detailed as possible so to avoid any confusion.

I believe I have posted this question in the proper category but if not, I apologize. 

1.  I have been experimenting with simple scenes such as forests.   I have trees, plants, flowers in the scene which render very nicely, but, images closest to the camera are blurred and look more like a water color painting rather than actual flowers and plants.  I don't know how to sharpen and give them a more realistic appearance. 

2. I am trying to set up the sun to shine directly from above or even other angles but every scene has the sun on the horizon shining at the camera like a spot light.   How do I adjust the position of the sun, that is, what prompts should I be using?  
-I've attached an image to illustrate my example. 

3. Under the "Advanced" tab are 2 settings.   -Guidance Scale  &  -Image sharpness.
As I increase the Guidance scale the details in my image get washed out and the scene is now farther away from what I want it to be. 
How exactly does the Guidance scale work?     The "Image sharpness" tool also seems to distort my image.  

4. If I want to send an image for printing, what setting/s should I take into consideration when rendering out the image?    That is, do I increase the DPI in a program like Photoshop, or, does Fooocus allow me to do that?  

Any suggestions will be very much appreciated.
Thank you.

![f 2 8 camera](https://github.com/lllyasviel/Fooocus/assets/165508031/ae8c5f1a-c92f-4ef3-878f-7c9ce9c195c9)

**Comment by ILikeToasters** [comment, marked as answer]
1. Without knowing more about prompt and settings/styles it can be hard to answer number one. Don't overdue the styles to start. I often disable all when I am trying to figure out why I am not getting the results I want since they do add more to the prompt and something may be causing whatever issue I am having. Amazing how one word can have an impact because of the association effect. 
2. Again hard to say not knowing the prompt but you can use things like noon, morning, and such. 
3. Guidance scale I won't really get into since if you want more detail it might be better to read up elsewhere on that. There are discussions on here that talk about it as well. Just be aware that Guidance scale in Fooocus is a little different than the others. Sharpness is exactly what it says. It does not change the image generation itself but just makes it more or less sharp. 
4. If you are going to want to do any printing you will need to upscale for sure and if going large may want to use other upscalers to go even higher than what Fooocus can do. As for the DPI that you won't do in Fooocus. 
Hopefully that helps.

**Reply by persius77** [reply]
Hi.  Thanks for replying.  
The prompts for me anyway remain a little confusing.  I say this because I originally started out by experimenting with Seaart and Leonardo.  When I looked at other peoples work (prompts) and used some as examples, nothing was working for me.  At this point I'm still at the trial and error stage but any guidance is always helpful. 

One thought.  under the tab "styles".   I have noticed that some people select 2 or 3 (or more) at one time.  What effect does that have on the rendered image?   Is there a simple reason for that because I think I'm overlooking something obvious. 

Thanks again.

**Reply by ILikeToasters** [reply]
Styles for the most part just add more keywords to your prompt. You can look in the log or metadata for the images if enable and it will show you what it adds. You can mix them but sometimes you may not get what you want if two styles are competing. In the Fooocus folder in the styles folder you can actually look up the json files for the styles to get a better understanding of what each one ads. This might help as well: https://youtu.be/Sr2yXOSexy0

**Comment by persius77** [comment]
Thanks so much.  At this point I still don't know what  I'm looking for unless I accidentally run into it, or, somebody shares a link.
Much appreciated.
